packages/playground/assets/js/playground.js

```diff
@@ -52,10 +66,11 @@
 	});
 
 	await client.isReady();
+	await client.goTo('/');
 
-	if (playground.pluginSlug) {
-		client.goTo('/wp-admin/plugins.php');
-	} else {
-		client.goTo('/wp-admin');
-	}
+	// if (playground.pluginSlug) {
+	// 	await client.goTo('/wp-admin/plugins.php');
+	// } else {
+	// 	await client.goTo('/wp-admin/');
+	// }
```

Comment:
@adamziel if I open the homepage or any other non admin page I see the admin bar. If I open `/wp-admin/` I get the login page, but I'm logged in when I click on another page.

I will investigate this tomorrow.